Clients/Apple/FitnessTracker/FitnessTrackerTests/MetricGraphInteractorTests.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  MetricGraphInteractorTests.swift
+//  FitnessTracker
+//
+//  Created by Alberto Chamorro on 06/02/2017.
+//  Copyright © 2017 OnsetBits. All rights reserved.
+//
+
+import Quick
+import Nimble
+import RxSwift
+@testable import FitnessTracker
+
+class MetricGraphInteractorTests: QuickSpec {
+    override func spec() {
+        describe("") {
```

Comment:
Add the following tests:
  * When the date matches the stored record,  the record should be returned
  * Given there is to records in the repository, when retrieving an interval that matches both records I would expect both records to be returned.
  * Given there is no records in the repository, an empty array should be returned.
  * Given there is n records in the repository, and m records are contained within a time interval I would expect m records to be returned
  * Not sure this needs tests for the time zone given the use cases. I'd say no.